<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadSettingForm</class>
 <widget class="QWidget" name="UploadSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#UploadSettingForm
{
	border-image: url(:/resource/images/common/Transparent_Black_80.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>144</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widgetBg" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1025</width>
       <height>431</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetBg
{
	background-color: rgb(255, 255, 255);
}
QListView::item::hover
{

}
</string>
     </property>
     <widget class="QWidget" name="widgetLeft" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>280</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widgetLeft
{
	background-color: rgb(149, 212, 244);
}</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 35px &quot;arial&quot;;</string>
       </property>
       <property name="text">
        <string>Upload Setting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/resource/images/upload_setting/Back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(149, 212, 244);
font: 70px &quot;Helvetica&quot;;</string>
      </property>
      <property name="text">
       <string>01</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(149, 212, 244);
font: 70px &quot;Helvetica&quot;;</string>
      </property>
      <property name="text">
       <string>02</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSplit">
      <property name="geometry">
       <rect>
        <x>346</x>
        <y>40</y>
        <width>575</width>
        <height>1</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(210, 210, 210);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelSplit_2">
      <property name="geometry">
       <rect>
        <x>346</x>
        <y>200</y>
        <width>575</width>
        <height>1</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(210, 210, 210);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpload">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>330</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: rgb(0, 0, 0);
	font: 70px &quot;arial&quot;;
	border:none
}
QPushButton:hover
{
	color: rgb(149, 212, 244);
	font: 70px &quot;arial&quot;;
	border:none
}
QPushButton:pressed
{
	color: rgb(149, 212, 0);
	font: 70px &quot;arial&quot;;
	border:none
}</string>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(150, 150, 150);
font: 20px &quot;arial&quot;;</string>
      </property>
      <property name="text">
       <string>Select Arduino Board</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxBoard">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>255</width>
        <height>35</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>-------------CoolMakers----------------</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CoolMakers Arduino Starter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CoolMakers Arduino Starter Plus</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxSerialPort">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>255</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(150, 150, 150);
font: 20px &quot;arial&quot;;</string>
      </property>
      <property name="text">
       <string>Select Serial Port</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUno">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>130</y>
        <width>59</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/upload_setting/MiniUno.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLeonardo">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>130</y>
        <width>59</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/upload_setting/MiniLeonardo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelTipToard">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>70</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please select your Arduino board.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;选择你使用的Arduino电路板&lt;/p&gt;&lt;p&gt;如果在列表中找不到你需要的型号 请查阅你使用的电路板的兼容Arduino型号&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#labelTipToard
{
	border-image: url(:/resource/images/upload_setting/tips.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelTipSerialPort">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>240</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please select your Arduino board serial port.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;选择你的Arduino所在的串口(Serial COM)&lt;/p&gt;&lt;p&gt;如果列表中默认的串口号不是你想要的 请手工修改&lt;/p&gt;&lt;p&gt;**当你把Arduino插入USB口时Mind+会自动监测到 但只会自动选择默认的第一个串口号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#labelTipSerialPort
{
	border-image: url(:/resource/images/upload_setting/tips.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
